pyproject.toml

```diff
@@ -19,6 +19,8 @@ classifiers = [
     "Typing :: Typed",
 ]
 dependencies = [
+    "h5py",
+    "pyvbmc",
```

Comment:
To get this to work nicely with FabSim3 `install_plugin` task we will need to pull these out in to a separate `requirements.txt` file and then have the `pyproject.toml` specify the `dependencies` property as dynamic and point to the file.